### PR TITLE
[OKTA-809878] Clarify SCIM update

### PR DESCRIPTION
**Pull request by annejuan-okta**
## Description:
- **What's changed?**  Add more info about how updates work in SCIM
- **Is this PR related to a Monolith release?**  Nope

### Resolves:

* [OKTA-809878](https://oktainc.atlassian.net/browse/OKTA-809878)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/scim/index.md](https://github.com/okta/okta-developer-docs/blob/d44a0248fa77eac2acfcc77d6b028f51d6258393/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fscim%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/773f3735-afb9-46ef-8dc3-97043148faba) | 81  | 84  | 86  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.8da8f21f-fa9a-4189-a49a-1ba4d8d70980?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/scim/index.md](https://github.com/okta/okta-developer-docs/blob/39d336196cf2d6695f31db04014663ea5844575e/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fscim%2Findex.md) | [87](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b9517209-3801-414b-8657-8243463304e8) | 81  | 88  | 90  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.4de578f7-b758-4b7f-8055-453809ceee66?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/scim/index.md](https://github.com/okta/okta-developer-docs/blob/2f7a7ad2affa054bd4b93fae6f64303fde156297/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fscim%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/dc071e12-fb8f-44cc-ba4c-ee77fdf07af8) | 80  | 87  | 90  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.f0a09bd0-642a-40fa-8ad8-218696d5c557?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/scim/index.md](https://github.com/okta/okta-developer-docs/blob/23e5b22ad498186f2605ddc9e259c10526091a64/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fscim%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/997f4f2c-7520-4bd1-8459-158c42d9c44d) | 80  | 87  | 90  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.39ca9ce8-54f9-4e4d-b3e3-560c6420e63d?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/scim/index.md](https://github.com/okta/okta-developer-docs/blob/b66993e60f70049ef7e62bfa1526322d65f90e5d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fscim%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5e0d565c-bdf1-46b2-8492-67e25c9b3bbb) | 80  | 87  | 90  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.e03dcb08-24f6-4571-bdec-81a061f2ba07?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by annejuan-okta** [comment]
Approved via chat with Scott on Slack.

File: packages/@okta/vuepress-site/docs/concepts/scim/index.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ meta:
...
 
 The goal of SCIM is to securely automate the exchange of user identity data between your company's cloud apps and any Service Providers (SP).
 
-Managing user lifecycles in your org is a fundamental business problem. Hiring employees is just the first step. You also need to provision apps needed for their job and enforce corporate security policies. Then you need to update user accounts as they advance through your company. At the end of their employment, you need to ensure that all access is quickly and thoroughly revoked across all apps. Handling all of this can be time-consuming and error-prone if done manually.
+Managing user lifecycles in your org is a fundamental business problem. Hiring employees is just step one. You also need to provision apps for their job and enforce corporate security policies. Then you need to update user accounts as they advance through your company. At the end of their employment, you need to ensure that all access is quickly and thoroughly revoked across all apps. Manually handling all of this can be time-consuming and error-prone.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/concepts/scim/index.md:20]: I think I'd leave this as "the first step" unless we're specifically going to number the remaining ones. I don't see any reference to additional steps in this doc, but maybe they're in a different one?

**Review by grahamsmith-okta — APPROVED** [review]
one question

**Review by grahamsmith-okta — APPROVED** [review]

**Pull request merged**